src/app.js

```
@@ -1 +1,70 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs/promises');
+
+async function copy(oldPath, newPath) {
+  if (!oldPath || !newPath) {
+    console.error('Source and destination file paths are required');
+  }
```

When the `copy` function is called without both paths you only log: `console.error('Source and destination file paths are required')`. The spec requires that errors be logged and propagated so callers/tests can observe them. After logging you should throw an Error (or return a rejected promise) to stop execution and indicate failure. (Checklist 3.1, 1.4)